README.md

```diff
@@ -135,12 +135,21 @@ All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metapa
 
 - DSC Composite Resources are not supported.
 - PSObjects like 'PSCredential' as parameters value not yet supported.
+- DSC requires PowerShell "Execution Policy" to be set appropriately for "LocalMachine" scope. If you see the error below, see [MODULES-2500](https://tickets.puppetlabs.com/browse/MODULES-2500) for more information.
```

Comment:
Is this better worded:

DSC requires the PowerShell `ExecutionPolicy` for the `LocalMachine` scope to be set to a less restrictive setting than `Restricted`. If you see the error below, see [MODULES-2500](https://tickets.puppetlabs.com/browse/MODULES-2500) for more information.